templates/firefox-ios.xliff

```diff
@@ -1084,6 +1084,18 @@
         <source>Remove</source>
         <note>When a user taps and holds on an item from the Recently Visited section, this label will appear indicating the option to remove that item.</note>
       </trans-unit>
+      <trans-unit id="FirefoxHomepage.Common.SitesCount.v101" xml:space="preserve">
+        <source>%d sites</source>
+        <note>Label showing how many sites there is in a search group. %d represents a number</note>
```

Comment:
Unfortunately we don't support proper plurals at the time :/
To work around this, could you rephrase to `Sites: %d`
More info about this [here](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#use-proper-plural-forms), at the bottom of the section.

Reply:
cc: @dnarcese just so you know, string will be "Sites: %d" instead